### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by lewisblake**
## Change Summary

- Update the index and intro
- Include new image better describing pyaerocom

## Related issue number

Related to https://github.com/metno/pyaerocom-tutorials/pull/13

## Checklist

* [x] Start with a draft-PR
* [x] The PR title is a good summary of the changes
* [x] PR is set to AeroTools and a tentative milestone
* [x] Documentation reflects the changes where applicable
* [x] Tests for the changes exist where applicable
* [x] Tests pass locally
* [x] Tests pass on CI
* [x] At least 1 reviewer is selected
* [x] Make PR ready to review

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -20,15 +20,19 @@
...
 def init_tutorials():
-    if not "conf.py" in os.listdir():
+    if "conf.py" not in os.listdir():
         raise FileNotFoundError("Wrong directory...")
 
-    if not TUTREPO in os.listdir():
+    if TUTREPO not in os.listdir():
         command = f"git clone {TUTURL}"
         print(command)
         subprocess.call(command, shell=True)
```
Comment (heikoklein) [inline docs/conf.py]: use `subprocess.run(command, shell=True, check=True)`
`subprocess.call` is from python <= 3.5
check=True raises exception on error

File: docs/conf.py [hunk 2]
```diff
@@ -20,15 +20,19 @@
...
-    if not TUTREPO in os.listdir():
+    if TUTREPO not in os.listdir():
         raise FileNotFoundError("Failed to clone pyaerocom-tutorials repo into pyaerocom/docs")
```
Comment (heikoklein) [inline docs/conf.py]: the `if` and `raise` is no longer needed if `git pull` raises an exception

File: docs/conf.py [hunk 3]
```diff
@@ -20,15 +20,19 @@
...
+    if TUTREPO in os.listdir():
```
Comment (heikoklein) [inline docs/conf.py]: `else:`

File: docs/conf.py [hunk 4]
```diff
@@ -20,15 +20,19 @@
...
+        command = "git pull"
+        print(command)
+        subprocess.call(command, shell=True)
```
Comment (heikoklein) [inline docs/conf.py]: `subprocess.run(command, shell=True, check=True)`

File: docs/conf.py (hunk 2 above)
Comment (lewisblake) [reply docs/conf.py]: It will raise `fatal`, which is not a python error but gets the point across.

File: docs/conf.py [hunk 5]
```diff
@@ -20,15 +22,16 @@
...
-    if not TUTREPO in os.listdir():
+    if TUTREPO not in os.listdir():
         command = f"git clone {TUTURL}"
         print(command)
-        subprocess.call(command, shell=True)
-    if not TUTREPO in os.listdir():
-        raise FileNotFoundError("Failed to clone pyaerocom-tutorials repo into pyaerocom/docs")
+        subprocess.call(command, shell=True, check=True)
+    else:
+        command = "git pull"
```
Comment (heikoklein) [inline docs/conf.py]: `command = f"cd {TUTREPO} && git pull"`

**Review by heikoklein — APPROVED** [review]

**Pull request merged**